hdr_html_canvas_element.md

```diff
@@ -266,167 +197,85 @@ The component signals are mapped to red, green and blue tristimulus values accor
-
-#### `srgb-linear`
-
-* Transfer function: identity
-
-* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
-
-* Linear light scaling: 1.0
-
-#### `srgb`
+### Color space conversions
 
-* Transfer function: See [srgb-linear](#srgb-linear)
+#### Background
 
-* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
+In general, application should avoid conversions between color spaces and maintain imagery in its original color space: conversions
+between color spaces are not necessarily reversible and do not necessarily result in the same image appearance. In particular,
+conversion of an HDR image to an SDR will result in a signification loss of information and an SDR image that is different from the
+SDR image that would have been mastered from the same source material. From that perspective, converting from HDR to SDR imagery is
+no different than converting RGBA images to 16-color pallette images.
 
-* Linear light scaling: 1.0
-
-_Note:_ The domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
-
-Also see note in the Issues section at the bottom about whether this space should be distinct from the existing `'srgb'` space.
-
-#### `rec2100-hlg`
-
-
-* Transfer function: The HLG Reference inverse OETF specified at Rec. ITU-R BT.2100, scaled
-
-```javascript
-  function connectionTransferFunction(x) {
-    const a = 0.17883277;
-    const b = 1 - 4*a;
-    const c = 0.5 - a * Math.log(4 * a);
-    const k = (Math.exp((0.75 - c) / a) + b) / 12;
-    if (x < 0) {
-      return 0;
-    } else if (x <= 0.5) {
-      return x * x / (3 * k);
-    } else if (x <= 1) {
-      return (Math.exp((v - c) / a) + b) / (12 *  k);
-    } else {
-      return 1 / scale;
-    }
-```
-
-_Note:_ the factor `k` is selected such that a signal of `0.75` results in a linear color value of 1 in the connection color space.
+Nevertheless, the HTML specification allows color space conversion in several scenarios, e.g., when [drawing images to a
+canvas](https://html.spec.whatwg.org/multipage/canvas.html#colour-spaces-and-colour-correction), [retrieving image data from a
+canvas](https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-getimagedata), among others being added). The conversions
+between predefined SDR color spaces are defined at <https://www.w3.org/TR/css-color-4/>, and this proposal similarly defines
+conversions for HDR color spaces.
 
-* Connection matrix: Identity
+These conversions fall into two broad categories:
 
-#### `rec2100-pq`
+* conversion between HDR color spaces
+* conversion between an HDR and an SDR color space (tone mapping)
```

Comment:
Tone mapping can also be the result of the HDR headroom of the display device being less than that of the mastering display, no?

Reply:
Yes. The text here however applies narrowly to the color space conversion that are covered by this explainer, which does not deal with what happens at the display.